scripts/exglobal_atmos_products.sh

```diff
@@ -0,0 +1,269 @@
+    echo "Reset nproc=${ncount}"
+  fi
+  inv=$(( ncount / nproc ))
+  rm -f "${DATA}/poescript"
+
+  last=0
+  for (( iproc = 1 ; iproc <= nproc ; iproc++ )); do
+    first=$((last + 1))
+    last=$((last + inv))
+    if (( last > ncount )); then (( last = ncount )); fi
+
+    # if final record of is u-component, add next record v-component
+    # if final record is land, add next record icec
+    # grep returns 1 if no match is found, so temporarily turn off exit on non-zero rc
+    set +e
+    # shellcheck disable=SC2312
+    ${WGRIB2} -d "${last}" "${tmpfile}" | grep -E -i "ugrd|ustm|uflx|u-gwd|land"
+    rc=$?
+    set_strict
+    if (( rc == 0 )); then  # Matched the grep
+      last=$(( last + 1 ))
+    fi
+    if (( iproc == nproc )); then
+      last=${ncount}
+    fi
+
+    # Break tmpfile into processor specific chunks in preparation for MPMD
+    ${WGRIB2} "${tmpfile}" -for "${first}":"${last}" -grib "${tmpfile}_${iproc}"
+    export err=$?; err_chk
+    input_file="${tmpfile}_${iproc}"
+    output_file_prefix="pgb2${grp}file_${fhr3}_${iproc}"
+    echo "${INTERP_ATMOS_MASTERSH} ${input_file} ${output_file_prefix} ${grid_string}" >> "${DATA}/poescript"
+
+    # if at final record and have not reached the final processor then write echo's to
+    # poescript for remaining processors
+    if (( last == ncount )); then
+      for (( pproc = iproc+1 ; pproc < nproc ; pproc++ )); do
+        echo "/bin/echo ${pproc}" >> "${DATA}/poescript"
+      done
+      break
+    fi
+  done  # for (( iproc = 1 ; iproc <= nproc ; iproc++ )); do
+
+  # Run with MPMD or serial
+  if [[ "${USE_CFP:-}" = "YES" ]]; then
+    "${HOMEgfs}/ush/run_mpmd.sh" "${DATA}/poescript"
+    export err=$?
+  else
+    chmod 755 "${DATA}/poescript"
+    bash +x "${DATA}/poescript" 2>&1 mpmd.out
+    export err=$?
+  fi
+  err_chk
+
+  # We are in a loop over downset, save output from mpmd into nset specific output
+  cat mpmd.out  # so we capture output into the main logfile
+  mv mpmd.out "mpmd_${nset}.out"
+
+  # Concatenate grib files from each processor into a single one
+  # and clean-up as you go
+  echo "Concatenating processor-specific grib2 files into a single product file"
+  for (( iproc = 1 ; iproc <= nproc ; iproc++ )); do
+    for grid in "${grids[@]}"; do
+      cat "pgb2${grp}file_${fhr3}_${iproc}_${grid}" >> "pgb2${grp}file_${fhr3}_${grid}"
+      rm  "pgb2${grp}file_${fhr3}_${iproc}_${grid}"
+    done
+    # There is no further use of the processor specific tmpfile; delete it
+    rm "${tmpfile}_${iproc}"
+  done
+
+  # Move to COM and index the product grib files
+  for grid in "${grids[@]}"; do
+    prod_dir="COM_ATMOS_GRIB_${grid}"
+    ${NCP} "pgb2${grp}file_${fhr3}_${grid}" "${!prod_dir}/${PREFIX}pgrb2${grp}.${grid}.${fhr3}"
+    ${WGRIB2} -s "pgb2${grp}file_${fhr3}_${grid}" > "${!prod_dir}/${PREFIX}pgrb2${grp}.${grid}.${fhr3}.idx"
+  done
+
+  # Create supplemental 1-degree grib1 output TODO: who needs 1-degree grib1 product?
```

Comment:
@WenMeng-NOAA 
Can you please confirm where 1-degree grib1 products are being used?
I do not see them on the NOMADS page for e.g. [2023-12-04 12z here](https://nomads.ncep.noaa.gov/pub/data/nccf/com/gfs/prod/gfs.20231204/12/atmos/)
There are no `RUN.tHHz.pgrb.1p00.fXXX` files on NOMADS for `gdas` or `gfs` cycles.

Reply:
@aerorahul The grib1 files might be used in the legacy verification system.  Since these products are not generated from the ops. gfs v16 any more, it should be safe that you remove this grib1 generation for gfs v17.

Reply:
removed in [ae16ddb](https://github.com/NOAA-EMC/global-workflow/pull/2110/commits/ae16ddb3f2a3c2859956e7fe29a7de1aa605b4de)